package/MDAnalysis/coordinates/base.py

```diff
@@ -1132,12 +1137,7 @@ def _read_next_timestep(self, ts=None):  # pragma: no cover
 
     def __iter__(self):
         self._reopen()
```

Comment:
I agree with the idea of using `rewind`, and it's better to be explicit. In practice, however, `rewind` is going through `_reopen` (at least in the `ProtoReader` class). Fixing the implicit use of `_reopen` should be a separate issue.

@kain88-de, since `rewind` is available in the base class I'd assume its use is safe.